src/main/java/com/example/gildedrose/GildedRoseService.java

```diff
@@ -0,0 +1,26 @@
+package com.example.gildedrose;
+
+import com.example.gildedrose.item.*;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+@Service
+public class GildedRoseService {
```

Comment:
**praise**: Nice job keeping Spring annotations from spreading throughout the application!